packages/paste-website/src/pages/components/topbar/index.mdx

```diff
@@ -0,0 +1,158 @@
+import {SidebarCategoryRoutes} from '../../../constants';
+import {Topbar} from '@twilio-paste/topbar';
+import {Menu, MenuItem, useMenuState, MenuBadge} from '@twilio-paste/menu';
+import {Badge} from '@twilio-paste/badge';
+import {Box} from '@twilio-paste/box';
+import {Input} from '@twilio-paste/input';
+import {SearchIcon} from '@twilio-paste/icons/esm/SearchIcon';
+import {
+  ProductSwitcher,
+  ProductSwitcherButton,
+  ProductSwitcherItem,
+  useProductSwitcherState,
+} from '@twilio-paste/product-switcher';
+import {LogoTwilioIcon} from '@twilio-paste/icons/esm/LogoTwilioIcon';
+import {ProductSegmentIcon} from '@twilio-paste/icons/esm/ProductSegmentIcon';
+import {ProductFlexIcon} from '@twilio-paste/icons/esm/ProductFlexIcon';
+import {ProductEmailAPIIcon} from '@twilio-paste/icons/esm/ProductEmailAPIIcon';
+import packageJson from '@twilio-paste/topbar/package.json';
+import DefaultLayout from '../../../layouts/DefaultLayout';
+import {getFeature, getNavigationData} from '../../../utils/api';
+import {defaultTopbarExample, emptyTopbarExample} from '../../../component-examples/TopbarExamples.ts';
+
+export default DefaultLayout;
+
+export const getStaticProps = async () => {
+  const navigationData = await getNavigationData();
+  const feature = await getFeature('Topbar');
+  return {
+    props: {
+      data: {
+        ...packageJson,
+        ...feature,
+      },
+      navigationData,
+    },
+  };
+};
+
+<NormalizedComponentHeader
+  categoryRoute={SidebarCategoryRoutes.COMPONENTS}
+  githubUrl="https://github.com/twilio-labs/paste/tree/main/packages/paste-core/components/topbar"
+  storybookUrl="/?path=/story/components-topbar--"
+  data={props.data}
+/>
+
+---
+
+<contentwrapper>
+
+<PageAside data={mdxHeadings} />
+
+<content>
+
+<LivePreview
+  scope={{
+    Topbar,
+    Box,
+    Input,
+    Menu,
+    MenuItem,
+    useMenuState,
+    MenuBadge,
+    Badge,
+    ProductSwitcher,
+    ProductSwitcherButton,
+    ProductSwitcherItem,
+    useProductSwitcherState,
+    LogoTwilioIcon,
+    ProductSegmentIcon,
+    ProductFlexIcon,
+    ProductEmailAPIIcon,
+    SearchIcon,
+  }}
+  noInline
+>
+  {defaultTopbarExample}
+</LivePreview>
+
+## Guidelines
+
+### About Topbar
+
+A Topbar is sticky to the top of the screen and consists of application-wide controls including:
+
+- [Account Switcher](/components/account-switcher)
+- Search [Input](/components/input)
+- [User Dialog](/components/user-dialog)
+- [Product Switcher](/components/product-switcher)
+
+## Examples
+
+Account-specific controls are aligned to the left, such as:
+
+- [Account Switcher](/components/account-switcher)
+- Account type indicators, such as subaccount [Badges](/components/badge)
+- Possible upgrade actions
+
+User-specific controls are aligned to the right, such as:
+
+- Search [Input](/components/input)
+- Support [Button](/components/button)
+- [Product Switcher](/components/product-switcher)
+- [User Dialog](/components/user-dialog)
```

Comment:
thought: I don't think this info should be under the `Examples` heading, but above it like the previous version of this page.


```suggestion
Account-specific controls are aligned to the left, such as:

- [Account Switcher](/components/account-switcher)
- Account type indicators, such as subaccount [Badges](/components/badge)
- Possible upgrade actions

User-specific controls are aligned to the right, such as:

- Search [Input](/components/input)
- Support [Button](/components/button)
- [Product Switcher](/components/product-switcher)
- [User Dialog](/components/user-dialog)


## Examples
```

Reply:
The content could all be combined into one section, potentially into a table, like:

A Topbar is sticky to the top of the screen and consists of application-wide controls including:

| Account-specific controls aligned to the left                                                      | User-specific controls aligned to the right                |
|----------------------------------------------------------------------------------------------------|------------------------------------------------------------|
| - Account Switcher - Account type indicators, such as subaccount Badges - Possible upgrade actions | - Search - Support button - Product Switcher - User Dialog |

Reply:
The table feels like squeezing too much new info together for me...

Reply:
fair. The first list essentially repeats the 2nd and 3rd still tho. Hard to parse what the difference is between the three lists.

Reply:
ok, combined them into something I think is more clear